junit-platform-commons/src/main/java/org/junit/platform/commons/util/ClassLoaderUtils.java

```diff
@@ -50,6 +56,43 @@ public static ClassLoader getDefaultClassLoader() {
 		return ClassLoader.getSystemClassLoader();
 	}
 
+	static URL toURL(String filePath) {
+		try {
+			return Paths.get(filePath).toUri().toURL();
```

Comment:
Replace `Paths.get` with `Path.of` factory - remove the then superseded `import java.nio.file.Paths;` statement as well.

Reply:
`Path.of` was introduced in Java 11 so we cannot use it here.

Reply:
My fault.

_We all play on 11, right?_

Reply:
Java 11 [java.nio.file.Paths](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Paths.html)'s Javadcoc contains the followin note:

> **API Note:**
It is recommended to obtain a `Path` via the `Path.of` methods instead of via the `get` methods defined in this class as this class may be deprecated in a future release.